packages/xpath/src/functions/xforms/node-set.ts

```diff
@@ -384,8 +386,39 @@ export const randomize = new NodeSetFunction(
 
 		const nodeResults = Array.from(results.values());
 		const nodes = nodeResults.map(({ value }) => value);
-		const seed = seedExpression?.evaluate(context).toNumber();
-
-		return seededRandomize(nodes, seed);
+		if (seedExpression === undefined) return seededRandomize(nodes);
```

Comment:
**No need to change here** (unless you agree/feel like it), but this tripped me up. I don't have a strong preference against single-line conditionals like this, but I'd generally prefer they're surrounded by a blank line so it's clear where they begin and end.

(But please feel free to ignore for now! I would definitely prefer to automate code style issues like this. I'm just mentioning it because it made me pause longer than I anticipated to make sure I understood the code.)